cmd/aries-agent-lambda/gorillamux.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
We started with https://github.com/awslabs/aws-lambda-go-api-proxy, but we think we can cut out gorillamux in favor of the standard library, to make it simpler.

Reply:
As @rolsonquadras mentioned, once you move this to [AFG-Ext](https://github.com/hyperledger/aries-framework-go-ext) repo as an independent module, then you don't have to worry about using gorillamux.